tests/test_arr_venue_v2.py

```diff
@@ -833,6 +833,34 @@ def test_june_cycle(self, client, openreview_client, helpers, test_client):
                 }
             )
         )
+
+        # Post duplicate (messy data)
+        openreview_client.post_invitation_edit(
```

Comment:
what are you testing here?

Reply:
This is how I replicated the error I was getting when running this script several times

Reply:
what is the error that you were getting? an API error?

Reply:
Yes when using live data I was getting an error that it was violating max replies

Reply:
I don't see here you are replicating the error, you increasing the max replies and posting a new registration. 

we need an assertion to make sure we get what we expect. is there a chance to run the script to reproduce the error?

Reply:
Without the changes in the PR, the script will run and it will try to copy data from the June cycle into the August cycle. The API will try to post 2 replies from `~Reviewer_ARRTwo1` into the August cycle which correctly has `maxreplies=1`